src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/WalletExceptionHandler.java

```diff
@@ -16,4 +16,11 @@ public String walletNotSavedHandler(WalletNotSavedException e) {
         log.error("WalletNotSavedException", e);
         return e.getMessage();
     }
+
+    @ExceptionHandler(WalletNotDeletedException.class)
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    public String walletNotDeletedHandler(WalletNotDeletedException e) {
+        log.error("WalletNotDeletedException", e);
```

Comment:
Please change it to:
log.error("WalletNotDeletedException: {}", e.getMessage() );

Reply:
This log changed.